examples/e2e-tests/kubernetes/release/features-off/addons-disabled.json

```diff
@@ -0,0 +1,56 @@
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes",
+      "kubernetesConfig": {
+        "addons": [
+          {
+            "name": "tiller",
+            "enabled": false
+          },
+          {
+            "name": "aci-connector",
+            "enabled": false
+          },
+          {
+            "name": "kubernetes-dashboard",
+            "enabled": false
+          },
+          {
+            "name": "rescheduler",
+            "enabled": false
+          }
+        ]
+      }
+    },
+    "masterProfile": {
+      "count": 5,

```

Comment:
For these tests let's build 1-master clusters to optimize test time and infra overhead.